homeassistant/components/panasonic_viera/manifest.json

```diff
@@ -1,7 +1,8 @@
 {
   "domain": "panasonic_viera",
-  "name": "Panasonic Viera TV",
+  "name": "Panasonic Viera",
   "documentation": "https://www.home-assistant.io/integrations/panasonic_viera",
-  "requirements": ["panasonic_viera==0.3.2", "wakeonlan==1.1.6"],
-  "codeowners": []
+  "requirements": ["panasonic_viera==0.3.5"],
+  "codeowners": [],
```

Comment:
Do you want to be codeowner?

Reply:
Wow! It would be great, but I didn't create the integration, so I don't know if that would be fair with all the other people who worked on it before.

Reply:
I mean, if there's no problem I'm fine with it :D

Reply:
Does that mean I'm assigned to `panasonic_viera`-related PRs?

Reply:
Yes, you'll be tagged on PRs and issues labeled with this integration. We would appreciate if you would want to be codeowner since we need help reviewing PRs and responding to issues.

Reply:
Please add yourself here and then run hassfest again.